devtron-labs/dashboard
# feat: Generic Empty component

# Description

To add a common empty state in
API Token List

Fixes [AB#1636](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/Devtron%20Team/Stories/?workitem=1636)

## Type of change


# How Has This Been Tested?
- Create a new cluster in full mode/EA Only mode
- Go to API Token List with no token available & check empty state
- 


# Checklist:
